test/Middleware/WhoopsErrorResponseGeneratorTest.php

```diff
@@ -155,4 +155,25 @@ public function testJsonContentTypeResponseWithJsonResponseHandler()
             $generator($error, $this->request->reveal(), $this->response->reveal())
         );
     }
+
+    public function testNonErrorStatusCodeIsSetTo500()
+    {
+        $error = new RuntimeException();
+
+        $this->whoops->getHandlers()->willReturn([]);
+        $this->whoops->handleException($error)->willReturn('WHOOPS');
+
+        $this->response->withStatus(StatusCode::STATUS_INTERNAL_SERVER_ERROR)->will([$this->response, 'reveal']);
+        $this->response->getBody()->will([$this->stream, 'reveal']);
+        $this->response->getStatusCode()->willReturn(StatusCode::STATUS_MOVED_PERMANENTLY);
```

Comment:
No, it's deliberately an error code below 400 to make sure `Utils::getStatusCode` does its job and returns the default `500` code. However I don't know if it should be tested here and just rely on Stratigility tests.